localization/english/dvg_flavor_bohemia_l_english.yml

```diff
@@ -4,7 +4,11 @@
  je_elbian_confederation_reason: "elbian_confederation"
  elbian_confederation_invalid: "Owns no Elbian homeland states"
  elbian_confederation_tt: "Country will become known as the Elbian Confederation"
- 
+ je_hre_external: "Centralising the Empires Borders"
+ je_hre_internal: "Centralising Bohemia?"
```

Comment:
I don't know about having a question mark here, and shouldn't it refer to centralizing the Empire? Maybe Centralizing the imperial diet or something

Reply:
This is probably me not quite understanding HRE/Bohemia but wouldn’t it be changing the laws of Bohemia and reforming them into a more autocratic state in line with a HRE? So it would be the empire as a whole right?